client/landing/subscriptions/components/settings/styles.scss

```diff
@@ -1,14 +1,13 @@
 @import "@automattic/color-studio/dist/color-variables";
 @import "@automattic/typography/styles/variables";
 
-$setting-item-margin-bottom: 20px;
+$setting-item-margin-bottom: 24px;
```

Comment:
24px could be directly inline if the variable is used just once?

Reply:
true! removed